examples/range-expressions/range_expressions.bal

```diff
@@ -27,7 +27,7 @@ public function main() {
         if (r is record {| int value; |}) {
             io:println(r.value);
         } else {
-            // `r` is `()`, implying the end of the iteration.
+            // `r` is `()` implying the end of the iteration.
```

Comment:
```suggestion
            // `r` being `()` implies the end of the iteration.
```

Reply:
I prefer the former. Specifically because this is a side by side explanation of the code.

Reply:
+1